src/node_crypto.cc

```diff
@@ -770,6 +809,8 @@ void SecureContext::AddRootCerts(const FunctionCallbackInfo<Value>& args) {
       BIO_free_all(bp);
       X509_free(x509);
     }
+
+    LoadExtraCaCerts(sc->env());
```

Comment:
It might be nice to add a `const char* filename` argument and change this to:
```c++
if (!extra_root_certs_file.empty())
  LoadExtraCaCerts(sc->env(), extra_root_certs_file.c_str());
```